tools/monorepo-utils/src/code-freeze/commands/version-bump/lib/update.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { readFile, writeFile, stat } from 'fs/promises';
+import { join } from 'path';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../../core/logger';
+
+/**
+ * Update plugin readme changelog.
+ *
+ * @param  tmpRepoPath cloned repo path
+ * @param  nextVersion version to bump to
+ */
+export const updateReadmeChangelog = async (
+	tmpRepoPath: string,
+	nextVersion: string
+): Promise< void > => {
+	const filePath = join( tmpRepoPath, `plugins/woocommerce/readme.txt` );
+	try {
+		const readmeContents = await readFile( filePath, 'utf8' );
+
+		const updatedReadmeContents = readmeContents.replace(
+			/= \d+\.\d+\.\d+ \d\d\d\d-XX-XX =\n/m,
+			`= ${ nextVersion } ${ new Date().getFullYear() }-XX-XX =\n`
+		);
+
+		await writeFile( filePath, updatedReadmeContents );
+	} catch ( e ) {
+		Logger.error( 'Unable to update readme changelog' );
```

Comment:
Since we catch a number of possible errors here I think logging the error message is wise, it will also help us debug issues in future.